tasks/setup.yml

```diff
@@ -34,15 +34,23 @@
 
 - name: "Rust | Define user variable for ssh use"
   set_fact:
-    fubarhouse_user: "{{ ansible_ssh_user }}"
```

Comment:
So fair enough it will be deprecated - however it isn't yet.

I'll be happy to accept this PR if you can add another task specifically for `ansible_user` without removing `ansible_ssh_user` - not everybody updates Ansible regularly and the tests are completely failing with this configuration with fairly recent versions of Ansible being used.

My main issue is the failing tests - I can update Ansible on 95% of them however I expect these tests still wouldn't pass.

Reply:
I have something like that already, but am trying to fix a few other situations in which it's not working.  Will keep testing and commit once ready.